Source/Core/Core/IOS/Network/IP/Top.cpp

```diff
@@ -254,8 +259,21 @@ static std::optional<DefaultInterface> GetSystemDefaultInterface()
                               get_addr(iface->ifa_broadaddr)};
     }
   }
+#else
+  u32 addr = 0;
+  u32 netmask = 0;
+  u32 gateway = 0;
+  // The thread prevent a stack corruption from JVM->AttachCurrentThread
+  std::thread([&] {
```

Comment:
You should be able to remove this since the last commit of PR #9078 added a different workaround for the problem.